tests/cli/test_cli.py

```diff
@@ -103,3 +105,33 @@ def test_cli_convert_ome_tiff(grid_layout, tmpdir):
     result = runner.invoke(cli, cmd)
     assert result.exit_code == 0, result.output
     assert "Converting" in result.output
+
+
+def test_cli_update_scale_metadata():
+    position_path = hcs_ref / "B" / "03" / "0"
+    with open_ome_zarr(position_path, layout="fov", mode="a") as input_dataset:
+        old_scale = input_dataset.scale
```

Comment:
This will permanently modify the reference file for the developers. See the NGFF tests for how to create a temporary copy.

Reply:
If you already ran this locally the pytest cache needs to be downloaded again.

Reply:
Sorry for polluting the fixtures! 

Thanks for flagging. Now fixed with your `_temp_copy`.